### PR TITLE
Disable test_bgp_queue mlnx 2700 T1

### PR DESCRIPTION
**Pull request by siqbal1986**
### Description of PR

The Mellanox 2700 T1 devices do not support sending BGP packets on queue 7. This test is being disabled.

Summary:
Fixes # (issue)
ADO 25587938
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [X] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [X] 202305
- [X] 202311

### CODE REVIEW

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 1]
```diff
@@ -116,9 +116,9 @@ bgp/test_bgp_multipath_relax.py:
 
 bgp/test_bgp_queue.py:
   skip:
-    reason: "Unsupported topology"
+    reason: "Unsupported topology or mellanox T1 asic"
     conditions:
-      - "topo_type in ['m0', 'mx']"
+      - "topo_type in ['m0', 'mx'] or ((topo_type in ['t1', 't1-lag', 't1-64-lag', 't1-56-lag'])and (platform in ['x86_64-mlnx_msn2700-r0', 'x86_64-mlnx_msn2700a1-r0']))"
```
Comment (bingwang-ms) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:121]: Do we see the issue on all t1 topo?

Comment (siqbal1986) [reply tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:121]: we see this issue on interfaces not part of lag. This is observed on T1 testbeds.

File: tests/common/plugins/conditional_mark/tests_mark_conditions.yaml [hunk 2]
```diff
@@ -116,9 +116,9 @@ bgp/test_bgp_multipath_relax.py:
 
 bgp/test_bgp_queue.py:
   skip:
-    reason: "Unsupported topology"
+    reason: "Unsupported topology or mellanox T1 asic"
```
Comment (bingwang-ms) [inline tests/common/plugins/conditional_mark/tests_mark_conditions.yaml:119]:
```suggestion
    reason: "Unsupported topology or mellanox T1-LAG asic"
```

**Review by bingwang-ms — APPROVED** [review]

**Comment by siqbal1986** [comment]
@yxieca can you please approve the casts so that this gets into other branches as well

**Comment by siqbal1986** [comment]
@StormLiangMS this is a test disable to reduce noise, please approve

**Comment by bingwang-ms** [comment]
MSFT ADO: 25587938
@StormLiangMS Can you help with the cherry-pick to 202305? Thanks

**Pull request merged**